src/SensioLabs/Melody/Melody.php

```diff
@@ -71,6 +71,14 @@ public function run($filename, array $arguments, RunConfiguration $configuration
 
         $cliExecutor($process, false);
     }
+    
+    /**
+     * @return string Returns the base-path used for temp and working files
+     */
+    public function getStoragePath()
+    {
+        return sprintf('%s/melody', sys_get_temp_dir());
```

Comment:
I initially used a property, but then decided to drop it, because it looked like a premature optimization and introduced IMO unnecessary noise.
Can re-store If you want the property.